acq_stat_reports/scripts/acq_stat_reports.py

```diff
@@ -0,0 +1,140 @@
+"""
+Generate acquisition statistics report.
+"""
+
+import argparse
+import json
+import os
+from pathlib import Path
+
+import ska_report_ranges
+from astropy import units as u
+from cxotime import CxoTime
+from ska_helpers import logging
+
+from acq_stat_reports import config as conf
+from acq_stat_reports import get_acq_info, get_data, make_acq_plots, make_html
+
+SKA = Path(os.environ["SKA"])
+
+
+logger = logging.basic_logger("acq_stat_reports", level="INFO")
+
+
+def get_parser():
+    parser = argparse.ArgumentParser(description=__doc__)
+    parser.add_argument(
+        "--datadir",
+        default=Path(SKA / "data" / "acq_stat_reports"),
+        help="Output directory (default is $SKA/data/acq_stat_reports)",
+        type=Path,
+    )
+    parser.add_argument(
+        "--webdir",
+        default=Path(SKA / "www" / "ASPECT" / "acq_stat_reports"),
+        help="Output directory (default is $SKA/www/ASPECT/acq_stat_reports)",
+        type=Path,
+        dest="output_dir",
+    )
+    parser.add_argument(
+        "--url",
+        help="URL root of the deployed page (default is /mta/ASPECT/acq_stat_reports/)",
+        default="/mta/ASPECT/acq_stat_reports/",
+    )
+    parser.add_argument(
+        "--start_time", help="Start time (default is NOW)", default=None
+    )
+    parser.add_argument(
+        "--days_back",
```

Comment:
Likewise this should be `--lookback`.  (See https://github.com/search?q=org%3Asot%20--lookback&type=code).

Reply:
This was removed. I don't think we want it anyway.